working_groups/tools/README.md

```diff
@@ -0,0 +1,24 @@
+# Tools Working Group
+
+The Tools working group of CHIPS Alliance covers the wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses around A/MS). 
+
+## Mission
+
+The mission of the Tools WG will be to:
```

Comment:
This section, the heart of the document, does a good job of covering our goals.  I'm trying to think it there is anything else we should be trying to achieve, but I think this covers our activities pretty well!  Nice job.

Reply:
Thanks!